app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,50 @@
+class OrganizationsController < ApplicationController
+  before_action :set_organization, only: [:show, :edit, :update]
+
+  def index
+    @organizations = Organization.all
+  end
+
+  def show
+  end
+
+  def new
+    @organization = current_user.organizations.new
+  end
+
+  def create
+    binding.pry
+    @organization = current_user.organizations.new organization_params
```

Comment:
This is not a blocker, but I generally prefer parenthesis around method calls in cases like this.